src/poll.ts

```diff
@@ -236,17 +250,36 @@ namespace Poll
 
 			return null;
 		}
+		
+		let userCount: number = 0;
+		for (let member of server.members.array())
+		{
+			if (member.user.presence.status === "online")
+			{
+				userCount ++;
+			}
+		}
+		
+		if (voicePoll)
+		{
+			userCount = voiceChannel.members.array().length;
+		}else 
```

Comment:
Please put brackets on their own line

Reply:
haha, it really takes mental effort to do this, so used to same line brackets (the superior format ;) ).

Reply:
lol, to each their own. The worst thing is a mix of both though, so just make sure to tolerate my formatting.